xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-api/src/main/java/org/xwiki/index/migration/R140300001XWIKI19571DataMigration.java

```diff
@@ -88,7 +88,9 @@ public XWikiDBVersion getVersion()
     public String getPreHibernateLiquibaseChangeLog() throws DataMigrationException
     {
         SessionFactory sessionFactory = this.hibernateStore.getSessionFactory();
+        String wikiId = getXWikiContext().getWikiId();
         try (SessionImplementor session = (SessionImplementor) sessionFactory.openSession()) {
+            this.hibernateStore.setWiki(session, wikiId);
```

Comment:
Why not simply `this.hibernateStore.setWiki(session)` ?

Reply:
That session is already initialized with the right wiki?

Reply:
Ok, it's computed internally inside `setWiki(session)`